Resources/Prototypes/Entities/Objects/Weapons/Guns/flare_gun.yml

```diff
@@ -1,39 +1,96 @@
 - type: entity
   name: flare gun
-  parent: BaseWeaponLauncher
+  parent: BaseItem
   id: WeaponFlareGun
   description: A compact, single-shot pistol that fires shotgun shells.
   components:
   - type: Sprite
     sprite: Objects/Weapons/Guns/Shotguns/flaregun.rsi
     layers:
-      - state: icon
+      - state: base
         map: ["enum.GunVisualLayers.Base"]
   - type: Item
     size: Small
     sprite: Objects/Weapons/Guns/Shotguns/flaregun.rsi
+  - type: ItemSlots
+    slots:
+      gun_chamber:
+        name: Chamber
+        startingItem: ShellShotgunFlare
+        priority: 1
+        whitelist:
+          tags:
+            - ShellShotgun
+            - ShellShotgunLight ## TODO: Add a risk of the gun blowing up if using non-light shotgun shells.
+  - type: ContainerContainer
+    containers:
+      gun_chamber: !type:ContainerSlot
+  - type: ChamberMagazineAmmoProvider
+    autoCycle: false
+    boltClosed: true
+    soundBoltClosed: /Audio/Weapons/Guns/Cock/revolver_cock.ogg
+    soundBoltOpened: /Audio/Weapons/Guns/Cock/revolver_cock.ogg
+    soundRack: /Audio/Weapons/Guns/Cock/revolver_cock.ogg
+  - type: Clothing
+    sprite: Objects/Weapons/Guns/Shotguns/flaregun.rsi
+    quickEquip: false
+    slots:
+    - Belt
+    - suitStorage
+  - type: Appearance
   - type: Gun
     fireRate: 8
     selectedMode: SemiAuto
     availableModes:
       - SemiAuto
     soundGunshot:
       path: /Audio/Weapons/Guns/Gunshots/flaregun.ogg
-  - type: BallisticAmmoProvider
-    whitelist:
-      tags:
-        - ShellShotgun
-    proto: ShellShotgunFlare
-    capacity: 1
-    soundInsert:
-      path: /Audio/Weapons/Guns/MagIn/shotgun_insert.ogg
+
+
+- type: entity
+  name: security shell gun
+  parent: [BaseItem, BaseSecurityContraband]
+  id: WeaponFlareGunSecurity
+  description: A modified flare gun designed to be used by security to launch non-lethal shotgun shells such as beanbags or tranquilizers. Can still fire lethal shells if neccesary.
```

Comment:
If it's suppose to be a non-lethal device, I'm not sure if it firing lethal shells is a good idea. What purpose is this meant to serve?

All new weapons need approval by either sloth or DrSmugleaf (https://github.com/space-wizards/space-station-14/issues/8524)

Reply:
Well... it's realistic as many irl flare guns are 12 gauge, and it's kind of a cool cheap improvised weapon.

Reply:
[Quoted from #design-discussion.](https://discord.com/channels/310555209753690112/1008709214006427689/1295768242975146015) 

Regular passengers will only have access to the regular flare gun (made out of plastic). You can fire shells in it just fine, but if you try to fire any shell other than a flare shell, the flaregun explodes and turns into a broken one, and we can maybe roll a 50% for the gun blowing up in their hand (clumsy system) or it “””working””” and shooting the shot, but one time.

Security will get a metal flaregun that just works with all shells. This is how they work in real life. 


Furthermore, the purpose of being able to fire lethal shells is both realism and for in an emergency self defense situation. I would assume giving out lethal sec shell guns is still considered a lethal weapon and thus code red only.
As for the weapon approval, [a maintainer said it should be fine unless somebody complains](https://discord.com/channels/310555209753690112/770682801607278632/1295803846450479155), so let me know if I need to still get review keeping in mind this is just a reskin of an already existing weapon while that weapon is being changed to no longer be lethal.